lib/functions/image/initrd.sh

```diff
@@ -88,8 +88,8 @@ update_initramfs() {
 		fi
 
 		# Convert to bootscript expected format, by calling into the script manually.
-		if [[ -f "${chroot_target}"/etc/initramfs/post-update.d/99-uboot ]]; then
-			chroot_custom "$chroot_target" /etc/initramfs/post-update.d/99-uboot "${initrd_kern_ver}" "/boot/initrd.img-${initrd_kern_ver}"
```

Comment:
How this will affect others?

Reply:
I have been building with this change for allwinner boards for quite some time now. Everything works as expected. No regression has been noticed due to this as the run-parts command runs the script now.

This just makes the behavior more consistent. Before this, the script that copies initrd to /boot/firmware was being triggered on cache miss, but was not being triggered on cache-hits. Now the script gets triggered correctly